### ISSUE TITLE
Viewer facelift - Layout 

### ISSUE THREAD

**Issue by sanmont3drepo** [3drepo/3drepo.io#3161]
### Description
The layout looks bad. We need to fix the colours and the button to match.

### Goals
- [ ] As a user I want to see the viewer consistent with the rest of v5

### Tasks

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/78047794/163210841-2799d840-c5f2-4719-96ff-a42b231f8cce.png">

**Circles**

Changes
<img width="77" alt="image" src="https://user-images.githubusercontent.com/78047794/163211539-fdeeac6c-504b-48f8-bbce-1289908daa42.png">

- [x] Make default background of circles white
- [x] Change the icon colour to primary.main on hover (background stays white)
- [x] Change the active state to secondary.main and the icon colour to primary.main
- [x] Align the circles vertically underneath the '3D' logo
- [x] Reduce to padding/margin between the circles to match the design
- [x] Move the cards a touch closer to the circles (I can't do this in css)

**Toolbar**

Changes
<img width="705" alt="image" src="https://user-images.githubusercontent.com/78047794/163211447-28dfdc57-be83-4a98-92cf-7278e7a7e85a.png">

- [x] Change background colour to secondary.main
- [x] Icons should be positioned horizontally centre inside the toolbar
- [x] Change the border-radius of the toolbar from 30px to 5px
- [x] Reduce spacing between icons slightly (I couldn't do this in css)
- [x] Add a 90% transparency to the background of the toolbar so that model is very slightly visible underneath
- [ ] Ensure the spacing between the bottom of the browser and the bottom of the toolbar is similar to my screenshot at the top of these comments (around 20px)

**Bugs in toolbar**
- [x]  Fix touch-area of a few buttons (in the horizontal bar) is reduced at the bottom. 
![image](https://user-images.githubusercontent.com/43547703/163345480-7e2e6431-cfb9-4bea-85bc-4f1b3825cfb0.png)
In other words, all the buttons with the small dash underneath cannot be clicked at the bottom like the other buttons (nor the mouse turns into a cursor). 

To give an idea, the red perimeter delimits the touchable area of the first two buttons
![image](https://user-images.githubusercontent.com/43547703/163346159-7b55fbb6-9e3d-4578-b78c-45530f0ea844.png)

This problem, however, exists in staging too for the same buttons
![image](https://user-images.githubusercontent.com/43547703/163345454-ced56280-ce39-460d-81f1-acda495f06d6.png)

### Related Resources
MockUp : `link to mockup`
